pkg/service/generateConfig/generateConfig.go

```diff
@@ -0,0 +1,81 @@
+package generateConfig
+
+import (
+	"os"
+	"sync"
+
+	"go.uber.org/zap"
+	"gopkg.in/yaml.v3"
+)
+
+var Emoji = "\U0001F430" + " Keploy:"
+
+type generatorConfig struct {
+	logger *zap.Logger
+	mutex  sync.Mutex
+}
+
+func NewGeneratorConfig(logger *zap.Logger) GeneratorConfig {
+	return &generatorConfig {
+		logger: logger,
+		mutex:  sync.Mutex{},
+	}
+}
+
+var config = `
+record:
+  path: ""
+  # mandatory
+  command: ""
+  proxyport: 0
+  containerName: ""
+  networkName: ""
+  delay: 5
+  passThroughPorts: []
+test:
+  path: ""
+  # mandatory
+  command: ""
+  proxyport: 0
+  containerName: ""
+  networkName: ""
+  testSets: []
+  globalNoise: |-
+    {
+      "global": {
+        "body": {},
+        "header": {}
+      },
```

Comment:
Please provide example of how we use global and test-set noise here. I found it hard to add noise in test because I didn't remember that I should use a list . So please provide an example in the form of comment at last in the yaml

Reply:
done!

Reply:
Example for test-set is missing

Reply:
You have specified "test-set-name" it is an example you have to provide real values like test-set-1

Reply:
done!